### DISCUSSION TITLE
Birdseye single camera

### DISCUSSION THREAD

**Discussion by DrSpaldo** [blakeblackshear/frigate#5437] [Ask A Question] [answered]
I would like to have a feed of only the latest camera that has had motion. So like birdseye, but, only one camera. Hopefully that makes sense...

Is this possible? or would this be a feature request?

**Comment by NickM-27** [comment, marked as answer]
That would be a feature request

**Reply by DrSpaldo** [reply]
Done https://github.com/blakeblackshear/frigate/issues/5441
